### DISCUSSION TITLE
Preferred maximum size for images

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#275] [Get Help] [answered]
Regarding images, on the page about [Figure Images](https://quire.getty.edu/documentation/figure-images/), it says that 1,800 pixels on the longest side works best for web and PDF. I want to let our photographers know how to formant the images.

Is there a preferred maximum file size for images?

**Comment by geealbers** [comment, marked as answer]
It's hard to say. When readers visit a page, their browser ultimately downloads every image on that page, so ideally you want file sizes to be as small as possible, to the point right before the average reader will notice that they look over-processed or pixelly in anyway. 

I can say that in our recent projects we have figures that are between 1mb (the largest I've ever want to go) and 100kb, with an avearge somewhere in the 450kb range. To get these file sizes we process all our JPGs down to an 80% quality.

Hope that helps.
